hathor/sysctl/init_file_loader.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional
+
+from hathor.sysctl.runner import SysctlRunner
+
+
+class SysctlInitFileLoader:
+    def __init__(self, runner: SysctlRunner, init_file: Optional[str]) -> None:
+        assert runner
+
+        self.runner = runner
+        self.init_file = init_file
+
+    def load(self) -> None:
+        if not self.init_file:
```

Comment:
I don't like this behavior of "silently not loading". I feel that this class should always expect an initialization file and fails if none is provided.

Reply:
Ok. I made it mandatory: baf31c3c81a310841b7e94c0266c14e2447e2de2

Reply:
run_node adjustment: f5b80e83c66ff705d6069d32a8845c596b7b969e

Reply:
Thanks!